website/packages/docs/src/pages/css-extraction-webpack.mdx

```diff
@@ -1,49 +1,145 @@
+
+Add `@compiled/babel-plugin-strip-runtime` to your Babel configuration, with `extractStylesToDirectory` set as an option. Set `source` to the folder your component library's source code is defined in, and `dest` to the folder your component library's build output is generated to.
+
+For example:
 
+```js
+// babel.config.js
 module.exports = {
-  module: {
-    rules: [
+  plugins: [
+    // This will handle all `token()` calls outside of
+    // Compiled usages
+    '@atlaskit/tokens/babel-plugin',
+
+    // ↓↓ Compiled should run last ↓↓
+    [
+      '@compiled/babel-plugin',
+      {
+        transformerBabelPlugins: ['@atlaskit/tokens/babel-plugin'],
+      },
+    ],
+    [
+      '@compiled/babel-plugin-strip-runtime',
       {
-        test: /\.(js|ts|tsx)$/,
-        exclude: /node_modules/,
-        use: [
-          { loader: 'babel-loader' },
-          {
-            loader: '@compiled/webpack-loader',
-+            options: {
-+              extract: true,
-+            },
-          },
-        ],
+        extractStylesToDirectory: {
+          source: 'src',
+          dest: 'dist',
+        },
       },
     ],
+  ],
+};
+```
+
+Your `dest` may vary depending on how you generate your `dist/` folder in your component library. For example, if you have mutliple `cjs` and `esm` distributions, those will each need a separate `dest` through Babel's environment-specific configuration.
+
+In each component library's `package.json` file, you will also need to add `"sideEffects": ["**/*.compiled.css"]`, to ensure that the Compiled stylesheets that are generated are exempt from Webpack's and Parcel's tree-shaking (which would delete the stylesheets!):
+
+```json
+{
+  "name": "@example/my-component-library",
+  // …
+  "dependencies": {
+    // Replace "version" with the latest version
+    "@compiled/react": "^version"
   },
-+  plugins: [
-+    new CompiledExtractPlugin(),
-+  ],
+  // …
+  "sideEffects": ["**/*.compiled.css"]
+  // …
+}
+```
+
+To test whether you've set up stylesheet extraction correctly, use your component library's usual build command (e.g. `yarn build`). Inside the `dist` folder, you should see some `.compiled.css` stylesheets generated, one for each `.tsx` file that uses `@compiled/react`.
+
+For example, if you have a file with the path `packages/design-system/button/dist/cjs/button.js` that uses Compiled, you can expect there to be a corresponding `packages/design-system/button/dist/cjs/button.compiled.css`.
+
+## Parcel stylesheet extraction
+
+If you are using a web app that uses Parcel, first install the [Parcel loader](https://compiledcssinjs.com/docs/installation#installation-methods).
+
+Then configure the `extract` plugin to `true` in your Compiled configuration. For example:
+
+```json
+// .compiledcssrc
+{
+  "addComponentName": true,
+  "extract": true, // <-- add this
```

Comment:
Ah, I should probably add this to `@atlaskit/css`?

Reply:
hmm i don't know why i kept `inlineCss` in there, i copied this from the jira compiled config 😅 

`addComponentName` is optional, i think it only works with the `styled` API

Reply:
Nevermind, this was already in that documentation, but good catch.